events/rateLimit.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * KlasaConsoleConfig#useColor must be
+ * enabled in order to use this piece.
+ */
+const { Event, Colors } = require('klasa');
+
+const HEADER = new Colors({ text: 'red' });
```

Comment:
You can also do

```javascript
const HEADER = new Colors({ text: 'red' }).format('[RATELIMIT]');
```

And then inside the method, put `HEADER`, so you don't format every time you hit ratelimits.

Reply:
Thank you for the suggestion! I'm gonna push a change right away. :3